wp-content/themes/pub/wporg-learn-2020/archive-lesson-plan.php

```diff
@@ -23,8 +191,8 @@
 
 		<hr>
 
-		<div class="row gutters between">
-			<div class="card-grid col-9">
+		<div class="lp-container row gutters between">
```

Comment:
`lp-container` doesn't have a lot of meaning, how about
```suggestion
		<div class="lp-archive-items row gutters between">
```